apps/cancel-request/translations/src/en/validation.json

```diff
@@ -2,9 +2,12 @@
     "cnc-main-applicant-full-name": {
         "required": "Enter the main applicant's full name",
         "maxlength": "Applicant's full name must be 3 and 250 characters",
-        "": "Applicant's full name must be 3 and 250 characters"
+        "minlength": "Applicant's full name must be 3 and 250 characters"
     },
     "cnc-main-applicant-dob": {
+        "required": "Enter the applicant's date of birth",
```

Comment:
This might need to say 'Enter the main applicant's date of birth' like we have for the other validation message? Can check with UCD :)